components/dashboard/src/App.tsx

```diff
@@ -113,6 +113,10 @@ function App() {
         window.location.href = `https://www.gitpod.io`;
         return <div></div>;
     }
+    if (window.location.pathname === '/chat') {
```

Comment:
We could also limit this behavior to `https://gitpod.io` only (i.e. not for staging or Self-Hosted):

```suggestion
    if (isGitpodIo() && window.location.pathname === '/chat') {
```